app/views/makers/new.erb

```diff
@@ -0,0 +1,17 @@
+<center>
+
+<br>
+<h1>Please sign up</h1>
+<br>
+
+<form action="/makers" method="post">
```

Comment:
see this article on best practice for html forms: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form

In particular you need to use the label tag and link the label semantically to the relevant input